packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -37,20 +37,6 @@ export const stableGCVersion: GCVersion = 3;
 /** The next version of GC Data, to bump to in case we need to regenerate all GC Data across all files. */
 export const nextGCVersion: GCVersion = 4;
 
-/**
- * This undocumented GC Option (on ContainerRuntime Options) allows an app to disable throwing an error when tombstone
- * object is loaded (requested), merely logging a message instead.
- *
- * By default, attempting to load a Tombstoned object will result in an error.
- */
-export const gcDisableThrowOnTombstoneLoadOptionName = "gcDisableThrowOnTombstoneLoad";
```

Comment:
This is used in `compatibilityConfiguration.ts`

Reply:
So let's keep the option and ditch the config key

Reply:
Updated tombstone to be enabled with sweep only. So, just setting `enableGCSweep` is enough.